pkg/lib/cockpit.js

```diff
@@ -117,6 +117,7 @@ function factory() {
     };
 
     cockpit.when = function when(value, fulfilled, rejected, updated) {
+        console.warn("cockpit.when is deprecated.");
```

Comment:
Do we really need to keep this? This has never been documented, and there are no external users. RPM and deb packaging already make sure that you can't install a newer cockpit-system with an older cockpit-bridge, and in Arch it's just a single package anyway. 

I.e. I'd vote for just killing it with :fire:

Reply:
Alright, works for me!